src/grafana/prometheus/docker/docker-compose.yml

```diff
@@ -8,21 +8,19 @@ services:
   # It could instead, use volume-mounts.
   # Use `docker exec -it prometheus /bin/sh` to logging in the container
   prometheus:
-    build: './prometheus'
+    image: prom/prometheus:v2.7.1
     container_name: 'prometheus'
     volumes:
       - 'scdf-targets:/etc/prometheus/'
+      - './prometheus-local-file.yml:/etc/prometheus/prometheus.yml'
     ports:
       - '9090:9090'
 
-  # The grafana container uses the image resulting from the build
-  # of `./grafana` which simply packs some configuration into the
-  # form of an image.
-  #
-  # This could instead be a volume-mounted container.
-  # Use `docker exec -it grafana /bin/bash` to logging in the container
+  # Grafana SCDF Prometheus pre-build image:
   grafana:
-    build: './grafana'
+    # To use a locally build grafana image, comment out the 'image:' and uncomment 'build:' section. Then run 'docker-compose up -d --build`
```

Comment:
is there a need to use a locally built one vs the public SCDF custom image?

Reply:
I've left those comments mostly for us, as a guideline how to debug  and test uncommitted changes.

Reply:
same reply as above